src/pudl/metadata/resources/eia860.py

```diff
@@ -500,6 +500,10 @@
                     "core_eia861__yearly_utility_data_misc",
                     "core_eia861__yearly_utility_data_nerc",
                     "core_eia861__yearly_utility_data_rto",
+                    "core_sec10k__quarterly_filings",
+                    "core_sec10k__quarterly_exhibit_21_company_ownership",
+                    "core_sec10k__quarterly_company_information",
+                    "out_sec10k__parents_and_subsidiaries",
```

Comment:
Exclude these because they don't have the same (annual) time frequency as the entity table.